### ISSUE TITLE
Color Contrast not high enough 

### ISSUE THREAD

**Issue by 00travelgirl00** [WordPress/twentynineteen#82]
The color contrast for post details and post navigation does not meet the AA WCAG standards. 

The color is #919191, paired with #ffffff the ratio is only 3.15

The font have to be at least 18px to match AA, but I think its better, to make post details and navigation darker. They have to be at least #757575 to match AA with a Ratio of 4.61. (But it still could be darker). 

Low Color Contrast #919191  (Ratio 3.15)
<img width="971" alt="bildschirmfoto 2018-10-17 um 00 51 34" src="https://user-images.githubusercontent.com/21318377/47052397-58d70e80-d1a8-11e8-80cd-081768a12ecc.png">

darker color #757575  (Ratio 4.61)
<img width="1064" alt="bildschirmfoto 2018-10-17 um 01 01 49" src="https://user-images.githubusercontent.com/21318377/47052399-5e345900-d1a8-11e8-9bbb-da249c88c611.png">

Tool for testing color contrast: https://contrastchecker.com/

**Comment by audrasjb** [comment]
Sure thing!
And I guess the issue has been fixed in #57 now 😃

**Comment by kjellr** [comment]
Hi @00travelgirl00 — Thanks for the report. As @audrasjb noted, that color was reported in #41 and fixed in #57. Marking this as closed.
